styles.css

```diff
@@ -0,0 +1,174 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+    margin: 0 auto;
+    text-align: center;
+    background-color: #444444;
+}
+
+nav {
+    padding: 20px;
+    background-color: #444444;
+    display: inline-block;
+    width: 100%;
+    height: 100px;
+    position: sticky;
+    top: 0;
+    z-index: 2;
+}
+
+a.anchor {
+    display: block;
+    position: relative;
+    visibility: hidden;
+}
+
+.navLinks a {
+    padding: 20px;
+    margin-right: 50px;
+    text-decoration: none;
+    border: 2px solid #00ffff;
+    color: #00ffff;
+}
+
+.hamburgerMenu div {
+    width: 35px;
+    height: 5px;
+    background-color: #00ffff;
+    border-radius: 12px;
+    margin: 6px auto;
+    display: none;
+}
+
+h1 {
+    font-size: 4vw;
```

Comment:
Font sizing using vw is a bit risky! Probably you will do this in the next task, but maybe using media queries is sufficient to change text size?